core/domain/user_jobs_continuous.py

```diff
@@ -164,13 +161,7 @@ def _get_most_recent_activity_commits(
 
         tracked_models_for_feedback = []
 
-        for ind, activity_model in enumerate(activity_models):
-            if activity_model is None:
```

Comment:
You should not delete this. Test it.

Reply:
I deleted this by mistake, sorry. Reverted!